### ISSUE TITLE
No sound when using Local MUNT

### ISSUE THREAD

**Issue by s9596362** [MiSTer-devel/Main_MiSTer#420]
I'm using MiSTer v210630. While I can get sound from using the local fluidsynth, when I switched to MUNT there's only silence (sound from the simulated SoundBlaster/Adlib still runs). I've tried restarting ao486 but still no sound from games that's supposed to support the MT32. When I switched to UDP mode and have my PC run MUNT, the games that run on ao486 does output the midi stream proper and I do get the music from my PC. But dragging my PC along wherever I bring my Mister defeats its portability.

I've search the internet for people that might have the same problem. Apparently, I'm alone in this.

Is there a way to downgrade midilink to test whether it's a problem with the latest version?

I'm quite a noob with MiSTer.

**Comment by sorgelig** [comment]
Please use MiSTer forum https://misterfpga.org/ for discussion.

**Comment by s9596362** [comment]
Thanks. Found someone with the exact same problem and some hints of how to fix it BUT I guess a permanent fix for it and to prevent other newbies from encountering the same problem in the future is that whoever that's in charged of the linux.img maintenance has to make sure that the latest version has a "mt32d" file in the sbin directory.

I fixed mine by renaming the mt32d_old file to mt32d. Hope that doesn't cause other problems within the linux environment. The suggested solution in the forum was to completely replace the linux.img on the miSTer with the one from the SD installer program though. FYI, I installed mine using the Mr Fusion method.

I guess now this issue can be closed.
